README.md

```diff
@@ -1384,6 +1399,53 @@ out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade s
    }
 }
 ```
+### Virtual energy service
+
+The service allows devices with output binary switch service or thermostat service to report accumulated energy consumption in case they do not have their own electric measurement or metering service. Thank to this service delivered energy is calculated based on manually provided power of a device. A device shall calculate energy at every relay state change, mode change or at least every interval value - 30 minutes by default. When service is added on a device, service meter_elec is created. service to send measurements.
+
+#### Service names
+
+`virtual_energy`
+
+#### Interfaces
+
+Type | Interface                | Value type | Description
+-----|--------------------------|------------|------------
+in   | cmd.set_interval         | int        | Interval in minutes for energy recalculation. Overwrites a default value.
+in   | cmd.add_device           | int_map    | Adds Virtual energy service to a selected device. The device shall be reporting energy consumption. Map of integers passed as an argument should provide energy consumption for every mode in watts.
```

Comment:
Integer map of course makes sense, but if I was to nitpick I would say that it should be a float map. Power and energy readings are reported as floats, so I see no reason why shouldn't we allow floats here as well even if it's a bit of an overkill. Also keys should not be string numbers, but mode names.